packages/jest-cli/src/get_node_handles.js

```diff
@@ -23,7 +23,14 @@ export default function collectHandles(): () => Array<Error> {
       Error.captureStackTrace(error, initHook);
     }
 
-    if (error.stack.includes('Runtime.requireModule')) {
+    if (
+      error.stack.includes('Runtime.requireModule') ||
+      (error.stack.includes('callAsyncCircusFn') &&
```

Comment:
shouldn't this be || instead of &&?

Reply:
I pushed a second commit extracting this into a function where I think the logic (that doesn't really work...) is easier to follow